dbt/include/bigquery/macros/adapters.sql

```diff
@@ -52,8 +52,10 @@
     {{ sql_header if sql_header is not none }}
 
     create or replace table {{ relation }}
+    {{ get_columns_spec_ddl() }}
```

Comment:
```suggestion
    {{ get_columns_spec_ddl() }}
    {{ get_assert_columns_equivalent(sql) }}
```
This creates a validation check that columns in a sql statement match the schema config